### ISSUE TITLE
groupby() doesn't show data

### ISSUE THREAD

**Issue by garawalid** [databricks/koalas#237]
DataFrame returned by `groupby()` can't show data in jupyter and terminal. 

This code returns `DataFrame[A: string, sum(__index_level_0__): bigint, sum(C): double, sum(D): double]`

```python
import numpy as np
import pandas as pd

from databricks import koalas as ks

kdf = ks.DataFrame({'A': ['foo', 'bar', 'foo', 'bar',
                          'foo', 'bar', 'foo', 'foo'],
                    'B': ['one', 'one', 'two', 'three',
                          'two', 'two', 'one', 'three'],
                    'C': np.random.randn(8),
                    'D': np.random.randn(8)})

print(kdf.groupby('A').sum())
```
Expected return

```python

pdf = pd.DataFrame({'A': ['foo', 'bar', 'foo', 'bar',
                          'foo', 'bar', 'foo', 'foo'],
                    'B': ['one', 'one', 'two', 'three',
                          'two', 'two', 'one', 'three'],
                    'C': np.random.randn(8),
                    'D': np.random.randn(8)})

print(pdf.groupby('A').sum())
```
```
A                      
bar  0.922431 -1.499922
foo -0.270957  1.153458
```

**Comment by HyukjinKwon** [comment]
Seems like we should implement repr.

**Comment by garawalid** [comment]
Fixed in version `0.2`
